tests/Extensions.Hosting.AsyncInitialization.Tests/Extensions.Hosting.AsyncInitialization.Tests.csproj

```diff
@@ -19,6 +19,8 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="FluentAssertions" Version="6.10.0"/>
+    <PackageReference Include="Moq" Version="4.18.0"/>
```

Comment:
Please don't introduce new dependencies.
The tests already use FakeItEasy, we don't need a separate mocking framework. And while I personally love FluentAssertions, this PR is not the place to introduce it. Just use the built-in xUnit assertion methods instead.